### DISCUSSION TITLE
Make Mastodon's user interface more accessible for people who rely on sign language

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#25943] [General]
This discussion is for sharing how Mastodon, and therefore the WebUI and both the app on Android and iOS, can become more accessible for people who rely on sign language.

There are projects like https://signdict.org/ or https://gebardenlernen.de that contain a lot of German Sign Language (DGS) signs that can be used under the license.

Whether there are similar overviews on an international level, e.g. for the English sign language, I unfortunately do not know.

To the question of why one should place value on people who are dependent on sign language, the answer is actually relatively clear: they would also like to use digital applications, and this is sometimes made very difficult for them.

If you make the first step here, then maybe this could also spread to other projects and general web projects and it would be so that many digital applications become more accessible for people who are dependent on sign language.

And that in turn would be a great success for society as a whole.

**Comment by bytebone** [comment]
I am most probably missing something here, but in what context to people who depend on sign language require this sign language to be placed on a digital screen? Am I wrong in thinking that sign language is primarily targeted at people who are hard or incapable of hearing, with reading capabilities unaffected / unrelated to this need? If you are trying to target people with vision impairments or dyslexia, I don't think sign language is the way to go.
